app/Controller/AttributesController.php

```diff
@@ -1497,11 +1426,11 @@ public function deleteSelected($id = false, $hard = false)
         $successes = array();
         foreach ($attributes as $a) {
             if ($hard) {
-                if ($this->__delete($a['Attribute']['id'], true)) {
+                if ($this->Attribute->__delete($a['Attribute']['id'], true)) {
```

Comment:
The parameters for __delete changed with the refactor, requires user to be passed along.

Reply:
Nice catch!